packages/ckeditor5-style/tests/ui/stylegridview.js

```diff
@@ -108,4 +116,140 @@ describe( 'StyleGridView', () => {
 			expect( grid.element.lastChild ).to.equal( grid.children.last.element );
 		} );
 	} );
+
+	describe( 'render()', () => {
+		beforeEach( () => {
+			grid.render();
+		} );
+
+		it( 'should register styleGridView children elements in #focusTracker', () => {
+			expect( grid.focusTracker._elements ).to.include( grid.children.first.element );
```

Comment:
We can't just access private properties of other classes like this. They are private for a reason: they may change often and that will break unrelated tests like this one. So either we do this properly by spying on `add()` or we don't do this at all because this is tested indirectly by keyboard navigation tests below.